src/backend/DocDownloaderRedisCache/src/main/java/ca/bc/gov/ag/courts/config/SecurityConfig.java

```diff
@@ -0,0 +1,54 @@
+package ca.bc.gov.ag.courts.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.security.web.SecurityFilterChain;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig {
+
+    @Value("${redisCache.auth.secret}")
+    private String password;
```

Comment:
According to the [Spring Documentation](https://docs.spring.io/spring-framework/reference/core/beans/annotation-config/value-annotations.html), it's quite unsafe to use `@Value` annotation because of the following:

> A default lenient embedded value resolver is provided by Spring. It will try to resolve the property value and if it cannot be resolved, the property name (for example ${catalog.name}) will be injected as the value.

Should we consider using `PropertySourcesPlaceholderConfigurer` instead?

Reply:
Add PropertySourcesPlaceholderConfigurer with profile spec.